source/fem-dem/fluid_dynamics_vans.cc

```diff
@@ -434,16 +480,33 @@ FluidDynamicsVANS<dim>::assemble_local_system_matrix(
     void_fraction_manager.void_fraction_locally_relevant,
     void_fraction_manager.previous_void_fraction);
 
-  scratch_data.reinit_particle_fluid_interactions(
-    cell,
-    this->evaluation_point,
-    this->previous_solutions[0],
-    this->void_fraction_manager.void_fraction_locally_relevant,
-    particle_handler,
-    this->dof_handler,
-    void_fraction_manager.dof_handler);
-
   scratch_data.calculate_physical_properties();
+
+  if (this->simulation_parameters.multiphysics.VOF)
+    {
+      scratch_data.reinit_particle_fluid_interactions(
+        cell,
+        this->evaluation_point,
```

Comment:
Hmm, that's something i'll have to doubel check. I thought we supported fully implicit coupling by evaluating everythign at the particle location using the current time-step values. We can leave it like that for now, but I'll have to revisit that